[2024-10-15T19:22:36.910+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-15T19:22:37.188+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_world_dag.hello_world_task2 scheduled__2024-02-04T00:00:00+00:00 [queued]>
[2024-10-15T19:22:37.288+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_world_dag.hello_world_task2 scheduled__2024-02-04T00:00:00+00:00 [queued]>
[2024-10-15T19:22:37.307+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-15T19:22:37.456+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): hello_world_task2> on 2024-02-04 00:00:00+00:00
[2024-10-15T19:22:37.565+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1374) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-15T19:22:37.582+0000] {standard_task_runner.py:63} INFO - Started process 1430 to run task
[2024-10-15T19:22:37.599+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'hello_world_dag', 'hello_world_task2', 'scheduled__2024-02-04T00:00:00+00:00', '--job-id', '269', '--raw', '--subdir', 'DAGS_FOLDER/example_hello_world.py', '--cfg-path', '/tmp/tmpi1m1cc9p']
[2024-10-15T19:22:37.654+0000] {standard_task_runner.py:91} INFO - Job 269: Subtask hello_world_task2
[2024-10-15T19:22:37.953+0000] {task_command.py:426} INFO - Running <TaskInstance: hello_world_dag.hello_world_task2 scheduled__2024-02-04T00:00:00+00:00 [running]> on host f9b6cc3118f7
[2024-10-15T19:22:38.589+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hello_world_dag' AIRFLOW_CTX_TASK_ID='hello_world_task2' AIRFLOW_CTX_EXECUTION_DATE='2024-02-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-04T00:00:00+00:00'
[2024-10-15T19:22:38.595+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-15T19:22:38.598+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-15T19:22:38.600+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'Hello World! Now I am triggered after the first task!'"]
[2024-10-15T19:22:38.663+0000] {subprocess.py:86} INFO - Output:
[2024-10-15T19:22:38.669+0000] {subprocess.py:93} INFO - Hello World! Now I am triggered after the first task!
[2024-10-15T19:22:38.671+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-15T19:22:38.673+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-15T19:22:38.760+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=hello_world_dag, task_id=hello_world_task2, run_id=scheduled__2024-02-04T00:00:00+00:00, execution_date=20240204T000000, start_date=20241015T192237, end_date=20241015T192238
[2024-10-15T19:22:38.862+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-15T19:22:38.928+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-15T19:22:38.935+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
